components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -1239,12 +1127,12 @@ interface IpfsService {
   GetNFTGatewayURL() => (string? url);
   GetGatewayURL() => (string? url);
 
-  SetGateway(string url) => (bool result);
-  SetNFTGateway(string url) => (bool result);
+  // SetGateway(string url) => (bool result);
+  // SetNFTGateway(string url) => (bool result);
 
   TranslateToGatewayURL(string url) => (string? translated_url);
-  TranslateToNFTGatewayURL(string url) => (string? translated_url);
-  ExtractIPFSUrlFromGatewayLikeUrl(string url) => (string? ipfs_url);
+//  TranslateToNFTGatewayURL(string url) => (string? translated_url);
+ ExtractIPFSUrlFromGatewayLikeUrl(string url) => (string? ipfs_url);
 
   ContentHashToCIDv1URL(array<uint8> content_hash) => (string? url);
 };
```

Comment:
Comments needs cleanup.
Also GetNFTGatewayURL and GetGatewayURL seem never referenced